src/scancode/cli.py

```diff
@@ -572,11 +583,25 @@ def scan_one(input_file, scanners, diag=False):
     return scan_result
 
 
-def save_results(files_count, scanned_files, format, input, output_file):
+def has_findings(active_scans, file_data):
+    for scan_name in active_scans:
+        if file_data.get(scan_name):
+            return True
+
+    return False
+
+
+def save_results(scanners, only_findings, files_count, scanned_files, format, input, output_file):
     """
     Save scan results to file or screen.
     """
-    # note: in tests, sys.sdtout is not used, but some io wrapper with no name attributes
+
+    if only_findings:
+        active_scans = [k for k, v in scanners.items() if v[0] and v[1]]
```

Comment:
To the point I was making above, it sounds like you are carrying a tuple in the value and have to resort to an index access. I usually try to stay away from indexed access as this does not make super explicit what is going on.